src/nft.sol

```diff
@@ -14,184 +14,29 @@
+        assetManager = IAssetManager(_assetManager);
     }
 
-    event Minted(address usr, uint tkn);
-
-    // --- Utils ---
-    function concat(bytes32 b1, bytes32 b2) pure internal returns (bytes memory) {
-        bytes memory result = new bytes(64);
-        assembly {
-            mstore(add(result, 32), b1)
-            mstore(add(result, 64), b2)
-        }
-        return result;
-    }
-
-     /**
-      * @dev Parses bytes and extracts a uint value
-      * @param data bytes From where to extract the index
-      * @return result the converted address
-      */
-    function bytesToUint(bytes memory data) internal pure returns (uint) {
-        require(data.length <= 256, "slicing out of range");
-        return abi.decode(data, (uint));
-     }
-
-     /**
-      * @dev Parses a uint and returns the hex string
-      * @param payload uint
-      * @return string the corresponding hex string
-      */
-     function uintToHexStr(uint payload) internal pure returns (string memory) {
-       if (payload == 0)
-         return "0";
-       // calculate string length
-       uint i = payload;
-       uint length;
-
-       while (i != 0) {
-         length++;
-         i = i >> 4;
-       }
-       // parse byte by byte and construct the string
-       i = payload;
-       uint mask = 15;
-       bytes memory result = new bytes(length);
-       uint k = length - 1;
-
-       while (i != 0) {
-         uint curr = (i & mask);
-         result[k--] = curr > 9 ? byte(55 + uint8(curr)) : byte(48 + uint8(curr));
-         i = i >> 4;
-       }
-
-       return string(result);
-     }
-
-    function equalBytes(bytes memory a, bytes memory b) internal pure returns (bool) {
-        if (a.length == b.length) {
-            for (uint i = 0; i < a.length; i++) {
-                if (a[i] != b[i]) {
-                    return false;
-                }
-            }
-            return true;
-        }
-        return false;
-    }
-
-    // --- NFT ---
-    function _checkAnchor(uint anchor, bytes32 data_root, bytes32 sig_root) internal view returns (bool) {
-        bytes32 doc_root;
-        (, doc_root, ) = anchors.getAnchorById(anchor);
-        if (data_root < sig_root) {
-           return doc_root == sha256(concat(data_root, sig_root));
-        } else {
-           return doc_root == sha256(concat(sig_root, data_root));
-        }
-    }
-
-    function tokenURI( uint token_id) external view returns (string memory) {
-        return string(abi.encodePacked(uri, "0x", uintToHexStr(uint(address(this))), "/0x", uintToHexStr(token_id)));
-    }
-
-  /**
-   * @dev Checks if the document is the latest version anchored
-   * @param data_root bytes32 hash of all data fields of the document which are signed
-   * @param next_anchor_id uint the next id to be anchored
-   */
-  function _latestDoc( bytes32 data_root, uint next_anchor_id)  internal view returns (bool) {
```

Comment:
why was all of this removed?

Reply:
we dont need these anymore since these no anchor repo